scm/doc/TechGuide/chap_hsd.tex

```diff
@@ -0,0 +1,37 @@
+\chapter{Hierarchical Physics Development}
+\label{chapter: Hierarchical_Physics_Development}
+
+Chapter 7 of the CCPP v6 Technical Documentation (\url{https://ccpp-techdoc.readthedocs.io/en/v6.0.0/}) provides a an overview of the tools supported by the SCM to faciliate hierarchical systems systems development (HSD)
+
+\section{Background}
+
+Developing and implementing a new physics parameterization for use in an operational setting requires extensive testing and evaluation. This is to ensure that new developments aren’t yielding unexpected results and that all computational considerations are being met. From a scientific perspective, this process should be incremental and hierarchical: starting with initial testing of a simple idealized case, then progressing to fully-coupled high-resolution global forecasts on high-performance computing systems.
+
+
+The CCPP-SCM provides developers working within CCPP compliant host models the ability to test their physics innovations without having to worry about the coupling to the dynamical core. This is a critical step in the model development hierarchy, providing insight on how an introduced physics change can modify the evolution of the internal physics state. However, there are still challenges, most notably the interactions between introduced changes and the other physics schemes in the suite.
+
+\section{CCPP Suite Simulator}
+
+\subsection{Overview}
+
+The CCPP Suite Simulator is a CCPP compliant physics scheme that provides the ability to turn off physical processes in a Suite Definition File (SDF), using namelist options. This simulator `piggybacks' on an existing SDF, replacing physics tendencies with data-driven tendencies. 
```

Comment:
So, "turning off" physics is accomplished by removing the scheme from the SDF, right? In other words, if the original scheme is included in the SDF, the scheme simulator does not automatically replace its tendencies -- you need to remove all schemes that are being simulated and put ccpp_scheme_simulator as many times as necessary in the SDF, right?

Reply:
This gets complicated when talking about suites with time-split process, but for suites with only process-split physics, to turn on/off physics you need to add the scheme to the end of the SDF and modify the namelist. Since the order of the processes doesn't matter when reconstructing the new state we can use the same SDF for all configurations. 

For time-split processes you need to add the suite-simulator scheme to the appropriate parts of the SDF, before and after each "active" scheme, and change the namelist. Since the order matter here we don't have fully functional on/off switches, as we do for process-split physics suites, and you need to create a new SDF for each configuration.

I'm adding examples to the documentation for both time-split and process-split physics suites.

For each example, I'm not proposing to remove anything from the SDF, just to add the suite-simulator scheme to the SDF. In theory once could remove the pieces being simulated from the SDF, but in practice there are secondary coupling issues that need to be addressed case-by-case.